examples/camera_streaming_http/camera_streaming_http.cc

```diff
@@ -41,7 +41,7 @@ constexpr char kIndexFileName[] = "/coral_micro_camera.html";
 constexpr char kCameraStreamUrlPrefix[] = "/camera_stream";
 
 HttpServer::Content UriHandler(const char* uri) {
-  if (StrEndsWith(uri, "index.shtml")) {
+  if (StrEndsWith(uri, "coral_micro_camera.html")) {
```

Comment:
Accessing  `http://<coralmicro_ip_address>/index.shtml` worked, but the README mentions explicitly that `/coral_micro_camera.html` serves the camera stream web page.

Reply:
Hi @atvrager ,

Thank you! I updated the PR. 

@hjonnala, I did not update the README since the referenced endpoint is working now.